src/hooks/useGetLoggedinUser.ts

```diff
@@ -0,0 +1,11 @@
+"use client";
+import { getCookies } from "@/utils/cookiesManager";
+import { useMemo } from "react";
+
+export const useGetLoggedInUser = () => {
+  const loginUser = useMemo(async () => {
+    const loginUserInfo = await getCookies("loginUser");
```

Comment:
can we use here enum because this string is using in two components ?

Reply:
fixed